timestamp,endpoint,parameters,error,status,misbehavior_detected,misbehavior_reason
2025-07-30T18:22:30.319176+00:00,http://localhost:8000/check-profile/,{'email': 'test@example.com'},"HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /check-profile/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d301d0>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:22:30.821567+00:00,http://localhost:8000/check-profile/,{'email': 'test@example.com'},"HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /check-profile/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d7c7d0>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:22:31.323664+00:00,http://localhost:8000/check-profile/,{'email': 'test@example.com'},"HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /check-profile/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d7d0d0>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:22:31.834628+00:00,http://localhost:8000/check-profile/,{'email': 'test@example.com'},"HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /check-profile/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d7dd30>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:22:32.336262+00:00,http://localhost:8000/check-profile/,{'email': 'test@example.com'},"HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /check-profile/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d7e6f0>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:22:32.838269+00:00,http://localhost:8000/check-profile/,{'email': 'test@example.com'},"HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /check-profile/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d7aae0>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:22:33.340176+00:00,http://localhost:8000/check-profile/,{'email': 'test@example.com'},"HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /check-profile/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d7e390>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:22:33.904432+00:00,http://localhost:8000/check-profile/,{'email': 'test@example.com'},"HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /check-profile/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d7dbe0>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:22:34.406318+00:00,http://localhost:8000/check-profile/,{'email': 'test@example.com'},"HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /check-profile/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d7cdd0>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:22:34.907778+00:00,http://localhost:8000/check-profile/,{'email': 'test@example.com'},"HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /check-profile/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d7ee10>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:22:35.410424+00:00,http://localhost:8000/check-profile/,{'email': 'test@example.com'},"HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /check-profile/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d7f950>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:22:35.912642+00:00,http://localhost:8000/check-profile/,{'email': 'test@example.com'},"HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /check-profile/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d7be00>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:22:36.414536+00:00,http://localhost:8000/check-profile/,{'email': 'test@example.com'},"HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /check-profile/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d7f890>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:22:36.940256+00:00,http://localhost:8000/check-profile/,{'email': 'test@example.com'},"HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /check-profile/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d7c3e0>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:22:37.441820+00:00,http://localhost:8000/check-profile/,{'email': 'test@example.com'},"HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /check-profile/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d7d400>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:22:37.943476+00:00,http://localhost:8000/check-profile/,{'email': 'test@example.com'},"HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /check-profile/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d7e660>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:22:38.468066+00:00,http://localhost:8000/check-profile/,{'email': 'test@example.com'},"HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /check-profile/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d80170>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:22:38.970097+00:00,http://localhost:8000/check-profile/,{'email': 'test@example.com'},"HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /check-profile/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d7ff50>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:22:39.471847+00:00,http://localhost:8000/check-profile/,{'email': 'test@example.com'},"HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /check-profile/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d7d880>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:22:39.995290+00:00,http://localhost:8000/check-profile/,{'email': 'test@example.com'},"HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /check-profile/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d7cb90>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:22:40.497138+00:00,http://localhost:8000/check-profile/,{'email': 'test@example.com'},"HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /check-profile/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d7f2f0>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:22:40.998979+00:00,http://localhost:8000/check-profile/,{'email': 'test@example.com'},"HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /check-profile/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d7fb30>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:22:41.524170+00:00,http://localhost:8000/check-profile/,{'email': 'test@example.com'},"HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /check-profile/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d80170>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:22:42.026841+00:00,http://localhost:8000/check-profile/,{'email': 'test@example.com'},"HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /check-profile/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d7f2c0>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:22:42.528814+00:00,http://localhost:8000/check-profile/,{'email': 'test@example.com'},"HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /check-profile/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d7ee10>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:22:43.054071+00:00,http://localhost:8000/check-profile/,{'email': 'test@example.com'},"HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /check-profile/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d7ec00>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:22:43.556013+00:00,http://localhost:8000/check-profile/,{'email': 'test@example.com'},"HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /check-profile/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d7ec30>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:22:44.058674+00:00,http://localhost:8000/check-profile/,{'email': 'test@example.com'},"HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /check-profile/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d804d0>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:22:44.560903+00:00,http://localhost:8000/check-profile/,{'email': 'test@example.com'},"HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /check-profile/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d810a0>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:22:45.062813+00:00,http://localhost:8000/check-profile/,{'email': 'test@example.com'},"HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /check-profile/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d7de20>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:22:45.564417+00:00,http://localhost:8000/check-profile/,{'email': 'test@example.com'},"HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /check-profile/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d7cf20>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:22:46.066317+00:00,http://localhost:8000/check-profile/,{'email': 'test@example.com'},"HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /check-profile/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d7c1d0>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:22:46.568334+00:00,http://localhost:8000/check-profile/,{'email': 'test@example.com'},"HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /check-profile/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d7eff0>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:22:47.070314+00:00,http://localhost:8000/check-profile/,{'email': 'test@example.com'},"HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /check-profile/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d801d0>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:22:47.572305+00:00,http://localhost:8000/check-profile/,{'email': 'test@example.com'},"HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /check-profile/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d81b50>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:22:48.074362+00:00,http://localhost:8000/check-profile/,{'email': 'test@example.com'},"HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /check-profile/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d7f350>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:22:48.576187+00:00,http://localhost:8000/check-profile/,{'email': 'test@example.com'},"HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /check-profile/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d7cf80>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:22:49.077772+00:00,http://localhost:8000/check-profile/,{'email': 'test@example.com'},"HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /check-profile/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d7dd60>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:22:49.579602+00:00,http://localhost:8000/check-profile/,{'email': 'test@example.com'},"HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /check-profile/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d7dc40>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:22:50.081367+00:00,http://localhost:8000/check-profile/,{'email': 'test@example.com'},"HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /check-profile/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d80140>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:22:50.583570+00:00,http://localhost:8000/check-profile/,{'email': 'test@example.com'},"HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /check-profile/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d82600>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:22:51.085256+00:00,http://localhost:8000/check-profile/,{'email': 'test@example.com'},"HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /check-profile/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d7ff50>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:22:51.587134+00:00,http://localhost:8000/check-profile/,{'email': 'test@example.com'},"HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /check-profile/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d7c2c0>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:22:52.088839+00:00,http://localhost:8000/check-profile/,{'email': 'test@example.com'},"HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /check-profile/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d7c050>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:22:52.590750+00:00,http://localhost:8000/check-profile/,{'email': 'test@example.com'},"HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /check-profile/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d7fdd0>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:22:53.092818+00:00,http://localhost:8000/check-profile/,{'email': 'test@example.com'},"HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /check-profile/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d81610>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:22:53.595213+00:00,http://localhost:8000/check-profile/,{'email': 'test@example.com'},"HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /check-profile/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d83020>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:22:54.097780+00:00,http://localhost:8000/check-profile/,{'email': 'test@example.com'},"HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /check-profile/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d7c980>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:22:54.600701+00:00,http://localhost:8000/check-profile/,{'email': 'test@example.com'},"HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /check-profile/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d7c3b0>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:22:55.102798+00:00,http://localhost:8000/check-profile/,{'email': 'test@example.com'},"HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /check-profile/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d7ec00>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:22:55.604274+00:00,http://localhost:8000/check-profile/,{'email': 'test@example.com'},"HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /check-profile/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d7e8d0>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:22:56.106014+00:00,http://localhost:8000/check-profile/,{'email': 'test@example.com'},"HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /check-profile/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d811c0>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:22:56.608164+00:00,http://localhost:8000/check-profile/,{'email': 'test@example.com'},"HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /check-profile/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d83a70>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:22:57.110064+00:00,http://localhost:8000/check-profile/,{'email': 'test@example.com'},"HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /check-profile/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d7dfa0>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:23:00.555464+00:00,http://localhost:8000/check-profile/,{'email': 'test@example.com'},"HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /check-profile/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d7de80>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:23:01.057246+00:00,http://localhost:8000/check-profile/,{'email': 'test@example.com'},"HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /check-profile/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d7c620>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:23:01.559298+00:00,http://localhost:8000/check-profile/,{'email': 'test@example.com'},"HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /check-profile/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d7f680>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:23:02.061193+00:00,http://localhost:8000/check-profile/,{'email': 'test@example.com'},"HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /check-profile/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d82150>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:23:02.564692+00:00,http://localhost:8000/check-profile/,{'email': 'test@example.com'},"HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /check-profile/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d88470>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:23:03.066988+00:00,http://localhost:8000/check-profile/,{'email': 'test@example.com'},"HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /check-profile/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d7f6b0>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:23:03.568730+00:00,http://localhost:8000/check-profile/,{'email': 'test@example.com'},"HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /check-profile/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d7e240>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:23:04.070348+00:00,http://localhost:8000/check-profile/,{'email': 'test@example.com'},"HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /check-profile/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d7cf20>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:23:04.572110+00:00,http://localhost:8000/check-profile/,{'email': 'test@example.com'},"HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /check-profile/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d7ff50>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:23:05.073716+00:00,http://localhost:8000/check-profile/,{'email': 'test@example.com'},"HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /check-profile/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d07170>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:23:05.576277+00:00,http://localhost:8000/check-profile/,{'email': 'test@example.com'},"HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /check-profile/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d89310>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:23:06.078137+00:00,http://localhost:8000/check-profile/,{'email': 'test@example.com'},"HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /check-profile/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d7d040>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:23:06.579780+00:00,http://localhost:8000/check-profile/,{'email': 'test@example.com'},"HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /check-profile/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d7f500>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:23:07.080897+00:00,http://localhost:8000/check-profile/,{'email': 'test@example.com'},"HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /check-profile/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d7f9e0>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:23:07.582745+00:00,http://localhost:8000/check-profile/,{'email': 'test@example.com'},"HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /check-profile/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d7f440>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:23:08.084706+00:00,http://localhost:8000/check-profile/,{'email': 'test@example.com'},"HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /check-profile/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d805f0>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:23:08.587044+00:00,http://localhost:8000/check-profile/,{'email': 'test@example.com'},"HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /check-profile/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d89df0>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:23:09.088517+00:00,http://localhost:8000/check-profile/,{'email': 'test@example.com'},"HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /check-profile/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d7f890>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:23:09.590189+00:00,http://localhost:8000/check-profile/,{'email': 'test@example.com'},"HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /check-profile/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d7f980>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:23:10.092052+00:00,http://localhost:8000/check-profile/,{'email': 'test@example.com'},"HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /check-profile/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d7d2e0>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:23:10.595131+00:00,http://localhost:8000/check-profile/,{'email': 'test@example.com'},"HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /check-profile/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d7d670>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:23:11.096890+00:00,http://localhost:8000/check-profile/,{'email': 'test@example.com'},"HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /check-profile/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d07170>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:23:11.598854+00:00,http://localhost:8000/check-profile/,{'email': 'test@example.com'},"HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /check-profile/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d8b440>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:23:12.100533+00:00,http://localhost:8000/check-profile/,{'email': 'test@example.com'},"HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /check-profile/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d7e8d0>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
